crowdgezwitscher/events/models.py

```diff
@@ -36,3 +36,21 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('events:detail', kwargs={'pk': self.pk})
+
+    def _is_ready_for_twitter(self):
+        """Checks that all fields required for getting tweets have some value."""
+        return all([self.twitter_account_names, self.twitter_hashtags, self.coverage_start, self.coverage_end])
```

Comment:
`build_twitter_search_query` sollte halbwegs robust sein, kommt damit auch mit Accounts XOR Hashtags klar.
Ich bin bisher aber davon ausgegangen, dass zur besseren Filterung Accounts UND Hashtags gesetzt werden. Diese Annahme ist womöglich falsch.
Kann ich heute Abend anpassen oder du machst es, wenn du Zeit hast (dann könntest du das überflüssige `'` am Zeilenende von Zeile 49 in den Models gleich noch entfernen).